src/main/java/seedu/address/model/person/TelegramHandle.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a Person's telegram handle in the address book.
+ */
+public class TelegramHandle {
+
+    public static final String MESSAGE_CONSTRAINTS = "Telegram handles should only contain alphanumeric characters "
+            + "and spaces, and it should not be blank";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Is it a good idea to standardise all Telegram handles to start with `@`?

Reply:
Not all Telegram accounts have handles, but I think we can standardise it to be so